Firestore/core/src/remote/grpc_connection.cc

```diff
@@ -119,8 +120,59 @@ class HostConfigMap {
 };
 
 HostConfigMap& Config() {
-  static HostConfigMap config_by_host_;
-  return config_by_host_;
+  static HostConfigMap config_by_host;
+  return config_by_host;
+}
+
+std::string GetCppLanguageToken() {
+  std::string cpp_version = [] {
```

Comment:
Since these are all literals, you could make this return `const char*` to save the copy.

Reply:
Done.